feat: disable fetching thumbnails if thumbnailSize is 0 (#14906)
electron/electron: docs/api/desktop-capturer.md

  * `types` String[] - An array of Strings that lists the types of desktop sources
    to be captured, available types are `screen` and `window`.
  * `thumbnailSize` [Size](structures/size.md) (optional) - The size that the media source thumbnail

>     should be scaled to. Default is `150` x `150`. Set width or height to 0 when not interested in

Reviewer: ```suggestion
    should be scaled to. Default is `150` x `150`. Set width or height to 0 when you do not need
```

Author: Done

>     the thumbnails. This save some processing like capturing the content of each window and screen

Reviewer: ```suggestion
    the thumbnails. This will save the processing time required for capturing the content of each window and screen.
```

Author: Done

>     plus the downscaling to the thumbnail size.

Reviewer: ```suggestion
```

Author: Done
